packages/types/src/codec/createType.spec.ts

```diff
@@ -232,4 +232,18 @@ describe('createType', () => {
       createType('{"_enum": {"A": null, "B": "u32", "C": null} }', 1).toJSON()
     ).toEqual({ B: 0 });
   });
+
```

Comment:
This is the unit test I added for the `Encoding for input doesn't match output, created 0x00 from 0x` issue above. Looks like this error appear even if the first argument of `createType` is not a double map type (`Vec<(BlockNumber,EventIndex)>`).

Reply:
Yes, `null` would be a funny one here. Not sure it would happen from storage, if that path does get taken we have some other issues there. (Which is the only place where `isPedantic` is currently used - if the decoding goes wrong, it goes horribly wrong there...)